docs/modules/ROOT/pages/documentation/payara-server/server-configuration/security/multiple-mechanism-in-ear.adoc

```diff
@@ -0,0 +1,40 @@
+[[multiple-httpauthenticationmechanism-ear]]
+= Multiple HttpAuthenticationMechanism in EAR
+
+_Since Payara Server 5.2020.3_
+
+Within the Security API specification, an `HttpAuthenticationMechanism` can be defined to retrieve the user credentials from the HTTP Request. These credentials are then used by the IdentityStore to validate the user. For an introduction, have a look at xref:https://blog.payara.fish/ee-security-jaspic-jacc-loginmodules-realms[this article]
+
+A restriction of the current Security API specification is that only 1 `HttpAuthenticationMechanism` can be defined within an application. This restriction also applies to an EAR file where you can have multiple Web application bundled.
+
+There is already an issue to address this restriction at the Security API project, but as an intermediate solution, We have introduced a new `web.xml` parameter within Payara so that you can define an `HttpAuthenticationMechanism` for each Web applications in an EAR artifact.
```

Comment:
```suggestion
There is already an issue to address this restriction at the Security API project, but as an intermediate solution, we have introduced a new `web.xml` parameter within Payara so that you can define an `HttpAuthenticationMechanism` for each Web applications in an EAR artifact.
```